bin/udb3.php

```diff
@@ -43,6 +44,7 @@
 
 /** @var \Knp\Console\Application $consoleApp */
 $consoleApp = $app['console'];
+$consoleApp->setCatchExceptions(false);
```

Comment:
I think we'll also have to fix this in the cdbxml service: https://github.com/cultuurnet/udb3-cdbxml-service/blob/master/bin/app.php#L35

Reply:
Fixed it in https://github.com/cultuurnet/udb3-cdbxml-service/pull/125